src/typings/Newsletter.d.ts

```diff
@@ -7,6 +7,24 @@ interface DraftNewsletter {
   standardMail: boolean;
 }
 
+interface DraftDirectLetter {
+  file: File | null;
+  contact_id: number | null;
+}
+
+interface DirectMailLog {
```

Comment:
wait what is this for?

Reply:
Older implementation - forgot to get rid of it - gone now!